tasks/main.yml

```diff
@@ -89,3 +89,10 @@
   delegate_to: '{{ item }}'
   when: item.split('.')[-1] == 'lxc'
   with_items: '{{ groups["all"] }}'
+          
+- name: Install sudo in container
+  raw: if ! hash sudo; then apt-get update -y && apt-get install -y sudo; fi
+  become: no
+  delegate_to: '{{ item }}'
+  when: item.split('.')[-1] == 'lxc'
+  with_items: '{{ groups["all"] }}'
```

Comment:
Can we can use the apt module here ? We used raw to install python, because modules won't work without python, but at this point python has been installed so ansible modules should work shouldn't they ?